### PR TITLE
Add skinny banner and logo section to /districts page

### PR DESCRIPTION
**Pull request by TurnerRiley**
Adds skinny banner and logo section to /districts page.

### Skinny banner (links to [/educate/districts/partners](https://code.org/educate/district/partners))
skinny banner

### Logo section
meet districts section

Responsive:
responsive

RTL:
RTL

## Links
Jira ticket: [items 2 and 3 from the ticket](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?assignee=60d62161f65054006980bd71&selectedIssue=ACQ-1687)
Figma: [here](https://www.figma.com/file/2CJiUNWSyspeHnLIt5Lw9z/District%2FAdmin-Landing-Page?type=design&node-id=1913-825&mode=design&t=clWkAFlOb2hYxocQ-0)

## Testing story
Local testing.

## Follow-up work
Item 4 from the ticket.

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/districts.haml [hunk 1]
```diff
@@ -111,6 +114,20 @@ theme: responsive_full_width
       %a.link-button{href: url_learn_more} Learn more
       %a.link-button.secondary{href: url_enroll_now, target: "_blank", rel: "noopener noreferrer"} Enroll now
 
+%section.bg-neutral-light
+  %h2.centered
+    Meet the Districts in our District Program
+  %p.centered
+    Over 100 school districts have already joined!
+  .logos.flex-container.justify-center.align-items-start.wrap
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/districts.haml]: You can use the `.logos-wrapper` class here; it should negate the need to add any custom styles, but you will need to set a `max-width` on the images ([see docs](https://docs.google.com/document/d/1wev8I2BQVViNXPuhLEXMpIAYSoAJb0kVoSRKaI3zEAk/edit#heading=h.buqozp3p1hoo))

**Review by kelbyhawn — COMMENTED** [review]
Looks good overall! I left a suggestion for the logos section, and can you double check that the logos work responsively? Using that `.logos-wrapper` class should automatically do it.

File: pegasus/sites.v3/code.org/public/districts.haml (hunk 1 above)
Comment (TurnerRiley) [reply pegasus/sites.v3/code.org/public/districts.haml]: Updated and added screenshots!

**Review by kelbyhawn — APPROVED** [review]
Thanks for updating! 🎉

**Pull request merged**